src/SmartComponents/SystemsTable/systemsHooks.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import helpers from '../helpers';
+import api from '../../api';
+import { Spinner } from '@patternfly/react-core';
+
+const useSystemsBulkSelect = (
+    id = '',
+    entities = {},
+    selectEntities,
+    bulkSelectDisabled,
+    isAddSystemNotifications
+) => {
+    const [ isLoading, setIsLoading ] = useState(false);
+
+    const setTitle = () => {
+        let title;
+        if (isLoading) {
+            title = <React.Fragment>
+                <Spinner size="sm" />
+            </React.Fragment>;
+        }
+
+        return { children: [ title ]};
```

Comment:
I believe this function returns `{ children: [undefined] }` when isLoading is false. Do you think that this will not lead to problems in the future?

Reply:
I changed it round a bit to only call this function when `isLoading === true`.